scrapi/base/__init__.py

```diff
@@ -171,36 +204,61 @@ def get_properties(self, result, property_list):
 
         properties = {}
         for item in property_list:
-            prop = (result.xpath('//dc:{}/node()'.format(item), namespaces=self.NAMESPACES) or [''])
-            prop += (result.xpath('//ns0:{}/node()'.format(item), namespaces=self.NAMESPACES) or [''])
-
-            properties[item] = prop if len(prop) > 1 else prop[0]
+            prop = (
+                result.xpath(
+                    '//dc:{}/node()'.format(item),
+                    namespaces=self.NAMESPACES
+                ) or ['']
+            )
+            prop += (
+                result.xpath(
+                    '//ns0:{}/node()'.format(item),
+                    namespaces=self.NAMESPACES
+                ) or ['']
+            )
+
+            if len(prop) > 1:
+                properties[item] = [self.copy_to_unicode(item) for item in prop]
+            else:
+                properties[item] = self.copy_to_unicode(prop[0])
```

Comment:
Yep! Wanted to have singular items show up as just one instead of one item inside a list - this is done to make things as generic as possible, and because items that are singular - publisher, type, etc, will always have just one element, and it'd be strange for them to show up as just one element in a list. 

We can discuss this though!